### PR TITLE
WRP-4945: Storybook - remove controls in docs tab

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
The controls from the Docs tab did not trigger in the rendered components

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
The controls from the Docs tab inside storybook and qa-storybook were removed.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-4945

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

File: samples/sampler/.qa-storybook/preview.js [hunk 1]
```diff
@@ -45,9 +46,15 @@ if (process.env.SKINS) {
 export const parameters = {
 	docs: {
 		container: DocsContainer,
-		page: DocsPage,
-		iframeHeight: 360,
 		inlineStories: false,
+		iframeHeight: ri.scaleToRem(600),
```
Comment (daniel-stoian-lgp) [inline samples/sampler/.qa-storybook/preview.js:50]:
```suggestion
		iframeHeight: ri.scaleToRem(300),
```

File: samples/sampler/.storybook/preview.js [hunk 2]
```diff
@@ -45,9 +46,14 @@ configureActions();
 export const parameters = {
 	docs: {
 		container: DocsContainer,
-		page: DocsPage,
-		iframeHeight: 360,
 		inlineStories: false,
+		iframeHeight: ri.scaleToRem(900),
```
Comment (daniel-stoian-lgp) [inline samples/sampler/.storybook/preview.js:50]:
```suggestion
		iframeHeight: ri.scaleToRem(450),
```

**Review by daniel-stoian-lgp — COMMENTED** [review]
we need to make the heights 2 times lower since agate is has a 24px/rem ratio

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**